docs/docs/troubleshooting.md

```diff
@@ -0,0 +1,32 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_title: Troubleshooting
+---
+### Summary
+
+The following page provides suggestions for common errors that may occur during firmware compilation. If the information provided is insufficient to resolve the issue, feel free to seek out help from the [ZMK Discord](https://zmkfirmware.dev/community/discord/invite).
+
+### Environment Variables not Defined
```

Comment:
The header changes are still missing here. Thanks.

Reply:
@petejohanson updated headers as requested.